core/client/components/gh-navitem.js

```diff
@@ -1,5 +1,11 @@
 var NavItemComponent = Ember.Component.extend({
     classNames: 'navigation-item',
+    classNameBindings: 'navItem.last:last-navigation-item',
+
+    attributeBindings:['getSort:data-sort'],
+    getSort: function () {
+        return this.get('navItem.sort');
+    }.property('navItem.sort'),
```

Comment:
We moved away from `property` in favor of `Ember.computed`, if you would my good man :D You can actually make this sort of behaviour really simple by doing
```js
attributeBindings: ['sort:data-sort'],
sort: Ember.computed.readOnly('navItem.sort')
```